tests/Agent/IntegrationTests/IntegrationTests/HttpClientInstrumentation/HttpClientInstrumentation.cs

```diff
@@ -163,9 +172,9 @@ public HttpClientInstrumentationTests_FW471(ConsoleDynamicMethodFixtureFW471 fix
     }
 
     [NetFrameworkTest]
-    public class HttpClientInstrumentationTests_FW48 : HttpClientInstrumentationTestsBase<ConsoleDynamicMethodFixtureFW48>
+    public class HttpClientInstrumentationTests_FW481 : HttpClientInstrumentationTestsBase<ConsoleDynamicMethodFixtureFW481>
```

Comment:
revert this, and add a new test target for `ConsoleDynamicMethodFixtureFWLatest`

Reply:
Will do